### PR TITLE
Add openbmc-sfw command to let users do a code update

### PR DESCRIPTION
**Pull request by charleshofer**
Signed-off-by: Charles P. Hofer <charles.hofer@ibm.com>

### CODE REVIEW

File: openbmc-sfw [hunk 1]
```diff
@@ -107,6 +178,24 @@ image_activate = subparsers.add_parser('activate', help='Activate input image id
...
+    '--reboot',
+    dest='reboot',
+    action='store_true',
+    default=False,
+    help='Set if the BMC should reboot after the update')
+image_update_auto.add_argument(
+    '--no-reboot',
+    dest='reboot',
+    action='store_false',
+    default=False,
```
Comment (geissonator) [inline openbmc-sfw]: Seems like we should make this the default

**Review by geissonator — COMMENTED** [review]
Good stuff Charlie!  Just please let me know on that reboot default.

File: openbmc-sfw (hunk 1 above)
Comment (charleshofer) [reply openbmc-sfw]: You want to not reboot by default? Or you want to reboot by default? Right now it's set to not reboot by default.

Comment (charleshofer) [reply openbmc-sfw]: You want to reboot by default or NOT reboot by default? Right now default is set to not reboot.

**Review by geissonator — APPROVED** [review]

**Pull request merged**